src/main/java/com/jcabi/s3/AwsBucket.java

```diff
@@ -98,6 +98,19 @@ public Ocket ocket(@NotNull(message = "key can't be NULL")
     }
 
     @Override
+    public boolean exists() {
+        final AmazonS3 aws = this.regn.aws();
+        boolean result = false;
+        try {
+            result = aws.doesBucketExist(this.bkt);
+        } catch (final AmazonServiceException ex) {
+            Logger.warn(this, ex.getMessage());
```

Comment:
what this is for? why not let it bubble up? exception swallowing is a terrible idea, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#exceptions